connect-id-example/src/main/java/com/telenor/connect/connectidexample/SignedInActivity.java

```diff
@@ -48,4 +44,39 @@ private void goToLogin() {
         finish();
     }
 
+    @Override
+    protected void onResume() {
+        super.onResume();
+        // Example usage of ConnectSdk.getValidAccessToken
+        ConnectSdk.getValidAccessToken(new AccessTokenCallback() {
+            @Override
+            public void success(String accessToken) {
+                Toast.makeText(SignedInActivity.this,
+                        "accessToken can now be used to access user sensitive resources",
+                        Toast.LENGTH_SHORT)
+                        .show();
+            }
+
+            @Override
+            public void unsuccessfulResult(Response response, boolean userWasCleared) {
```

Comment:
The parameter "userWasCleared" could be misinterpreted as meaning something like "user was granted security clearance". Could perhaps be renamed to something like "userDataRemoved".

Reply:
Sounds good 👍📝